terminus-ui/radio-group/src/radio-group.component.scss

```diff
@@ -51,8 +51,10 @@
 .c-radio--sublabel {
```

Comment:
Weird.. just noticed this original class is using `--` where it should be using `__`.. guess I should do a classes scrub at some point 😞